server/controllers/v1/tests/user.test.ts

```diff
@@ -234,20 +236,9 @@ describe('v1 - User Controller', () => {
       await expect(uContr.confirmEmail(rawToken)).resolves.toBeFalsy();
     });
 
-    test('Ensure nothing happens if the new password is invalid', async () => {
```

Comment:
why remove?

Reply:
I removed this since this was a test for checking if a password matches the confirmation password. Since, I will have to add that confirmation validation back in, I will also add this test back in.